mesa/experimental/jupyter_viz.py

```diff
@@ -133,28 +133,37 @@ def function(viz):
     return function
 
 
-def make_user_input(user_input, k, v):
-    if v["type"] == "SliderInt":
+def make_user_input(user_input, options):
```

Comment:
I initially included the name ("k") because I figured it might be used in the future. Looking at the code again, I think the name can be used as a fallback label instead of the current generic "label".

Reply:
Ah, ok, that makes sense; I'll revise.

Should the method throw an exception if the type is not supported? If so, does `ValueError` seem appropriate to you?

Reply:
@rht updated with name fallback logic, exception for unsupported type, and preliminary unit test for this method